Release 2022.06.2 (#3431)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2022/index.md

## June

### Weekly release 2022.06.2

| Change | Expected in Preview Orgs |
|--------------------------------------------------------------------------|--------------------------|

> | [Reset Factors endpoint includes new optional request parameters](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |

Reviewer: ```suggestion
| [Reset Factors endpoint includes new a optional request parameters](#reset-factors-endpoint-includes-new-a-optional-request-parameters)                      | June 23, 2022            |
```

| [Bugs fixed in 2022.06.2](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |

> #### Reset Factors endpoint includes new optional request parameter

Reviewer: ```suggestion
#### Reset Factors endpoint includes a new optional request parameter
```

> The `/reset_factors` endpoint has a new optional request parameter for the Reset Factor lifecycle operation. You can now remove the phone factor (for example: SMS/Voice) as both a recovery method and a factor with one call by setting the `removeRecoveryEnrollment` parameter to true when making a POST request to the `/reset_factors` endpoint (`/users/${userId}/lifecycle/reset_factors`). <!--OKTA-500168-->

Reviewer: ```suggestion
The `/reset_factors` endpoint has a new optional request parameter for the [Reset Factor lifecycle operation](/docs/reference/api/users/#reset-factors). You can now remove the phone factor (for example: SMS/Voice) as both a recovery method and a factor by setting the `removeRecoveryEnrollment` parameter to true when making a POST request to the `/reset_factors` endpoint (`/users/${userId}/lifecycle/reset_factors`). <!--OKTA-500168-->
```
I'm thinking that there should be a link to the Reset Factor API operation? You can remove the link if you think it's not necessary.

Author: I agree, for the feature update, to add the link.

#### Bugs fixed in 2022.06.2

* When an OAuth 2.0 client was created with a missing JWKS RSA modulus value (`n` parameter), the JWKS validation failed. (OKTA-424664)

> * Using the JSON Web Key API or the Client Credentials API to manage client credentials didn't fire System Log events. (OKTA-494619)

Reviewer: ```suggestion
* When the JSON Web Key API or the Client Credentials API was used to manage client credentials, the event wasn't triggered in the System Log. (OKTA-494619)
```
